scripts/metricsgen/metricsgen_test.go

```diff
@@ -158,7 +158,7 @@ func TestParseMetricsStruct(t *testing.T) {
 						TypeName:   "Counter",
 						FieldName:  "myCounter",
 						MetricName: "my_counter",
-						Labels:     "label1, label2",
+						Labels:     "\"label1\",\"label2\",",
```

Comment:
```suggestion
						Labels:     `"label1","label2"`,
```

not sure if the trailing comma was intentional.

Reply:
It was, sort of a lazy way to not have to do the bounds checking since `format` takes care of stripping it for you.

Reply:
using `strings.Join` is a better solution though.